styles/avatar.css

```diff
@@ -0,0 +1,122 @@
+.avatar {
+  position: relative;
+}
+
+.avatar-sm {
+  height: 3rem;
+  width: 3rem;
+  border-radius: 100%;
+}
+
+.avatar-md {
+  height: 5rem;
+  width: 5rem;
+  border-radius: 100%;
+}
+
+.avatar-lg {
+  height: 6rem;
+  width: 6rem;
+  border-radius: 100%;
+}
+
+.avatar-lg-online {
+  position: absolute;
+  background-color: green;
+  bottom: 10;
+  right: 0;
+  height: 1rem;
+  width: 1rem;
+  border-radius: 100%;
+  border: 2px solid white;
+}
+
+.avatar-md-online {
+  position: absolute;
+  background-color: green;
+  bottom: 6;
+  right: 0;
+  height: 1rem;
+  width: 1rem;
+  border-radius: 100%;
+  border: 2px solid white;
+}
+
+.avatar-sm-online {
+  position: absolute;
+  background-color: green;
+  bottom: 0;
+  right: 0;
+  height: 0.8rem;
+  width: 0.8rem;
+  border-radius: 100%;
+  border: 2px solid white;
+}
+
+.avatar-lg-offline {
+  position: absolute;
+  background-color: red;
```

Comment:
use your color variables instead of color names here and other places in this file wherever red and white colors are used.